images/ad-server/install-packages.sh

```diff
@@ -17,22 +16,27 @@ case "${install_packages_from}" in
     samba-nightly)
         # unset version suffix for nightly builds
         samba_version_suffix=""
-        get_custom_repo "https://artifacts.ci.centos.org/samba/pkgs/master/fedora/samba-nightly-master.repo"
+        get_custom_repo "https://artifacts.ci.centos.org/samba/pkgs/master/centos/samba-nightly-master.repo"
     ;;
     custom-repo)
         get_custom_repo "${install_custom_repo}"
     ;;
 esac
 
-dnf install --setopt=install_weak_deps=False -y \
+dnf --enablerepo=crb --enablerepo=resilientstorage \
+    install --setopt=install_weak_deps=False -y \
     findutils \
     python-pip \
     python3-jsonschema \
     python3-samba \
     python3-pyxattr \
-    "samba-dc${samba_version_suffix}" \
-    procps-ng \
-    /usr/bin/smbclient
+    "samba${samba_version_suffix}" \
+    "samba-client${samba_version_suffix}" \
+    "samba-winbind${samba_version_suffix}" \
+    "samba-winbind-clients${samba_version_suffix}" \
+    tdb-tools \
+    "ctdb${samba_version_suffix}"
```

Comment:
It's possible I'm reading this wrong, but it appars to replace the samba-dc package with samba file server packages -- effectively turning the ad-server image into a copy of the (file) server image. I'm very confused.

Reply:
@phlogistonjohn ,  I think I did the change initially by copying the files from @anoopcs9 's patch which in fact touched the file server image  😬  ...